portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import java.io.IOException;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/greeting")
+public class GreetingServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // ArrayList holds facts about myself
+    ArrayList<String> messages = new ArrayList<String>();
```

Comment:
A good rule of thumb for when to use comments is when we want to explain why we do something that isn't immediately obvious, or if the function is really long and we want to summarize it. In this instance, the comment "ArrayList holds facts about myself" can probably be taken out and you could rename the `messages` variable to `funFacts`. 

Similarly below, the comment of "function returns JSON by using Gson" is not really helpful because the function name already says basically the same thing, so you can take that out.